src/ai/backend/manager/api/vfolder.py

```diff
@@ -2424,6 +2424,15 @@ async def clone(request: web.Request, params: Any, row: VFolderRow) -> web.Respo
     if source_proxy_name != target_proxy_name:
         raise InvalidAPIParameters("proxy name of source and target vfolders must be equal.")
 
+    if row["group"] is not None:
+        ownership_type = "group"
+        if not request["is_admin"]:
+            raise GenericForbidden("no permission")
+    else:
+        ownership_type = "user"
+    if ownership_type not in allowed_vfolder_types:
+        raise InvalidAPIParameters(f"{ownership_type}-owned vfolder is not allowed in this cluster")
+
```

Comment:
We already handle `allowed_vfolder_types` in `query_accessible_vfolders()` method.
As you can see in `query_accessible_vfolders()`, we assume that the system has ownership type `user` as a default value.
So how about removing this part?

Reply:
Thanks! I will remove code about allowed_vfolder_types.
And I have a question about checking admin code.
When creating a vfolder, if user is not an admin, user cannot create a project vfolder, so there is no need to check whether user is an admin or not in the clone function, right?
Or is it correct to explicitly check whether or not user is an admin for any unknown risks?

Reply:
As long as it is unable to clone a vfolder as a project folder, I think we dont need to double check it.

Reply:
Thanks to reviewing!
I will remove that code while keeping it working.